pkg/tsdb/cloudwatch/log_actions.go

```diff
@@ -218,6 +219,11 @@ func (e *cloudWatchExecutor) handleStartQuery(ctx context.Context, logsClient cl
 	model *simplejson.Json, timeRange backend.TimeRange, refID string) (*data.Frame, error) {
 	startQueryResponse, err := e.executeStartQuery(ctx, logsClient, model, timeRange)
 	if err != nil {
+		var awsErr awserr.Error
+		if errors.As(err, &awsErr) && awsErr.Code() == "LimitExceededException" {
+			plog.Info("executeStartQuery limit exceeded", "err", awsErr)
```

Comment:
```suggestion
			plog.Error("executeStartQuery limit exceeded", "error", awsErr)
```

Reply:
Not sure about the `Info` vs `Error`. My thinking was that if I see `Error` in the logs it's something unexpected that should be dealt with and fixed. But this is just something that can happen and not much you can do with it (you can rise the Cloud Watch limits through support request but probably not to super high number so it won't really fix this) so I thought Info makes more sense.

Reply:
Or maybe `Warning` is more appropriate.

Reply:
So maybe `Debug` then? Even with a warning, there's nothing you can do about it